datahub-web-react/src/app/entity/shared/containers/profile/utils.ts

```diff
@@ -27,6 +27,12 @@ export function getDataForEntityType<T>({
             customProperties,
         };
     }
+    if (anyEntityData.tags) {
```

Comment:
Nice! Just please make sure this works for datasets, charts, dashboards as well.